src/tests/ftest/control/version.py

```diff
@@ -22,32 +22,56 @@ class DAOSVersion(TestWithServers):
     def __init__(self, *args, **kwargs):
         """Initialize a DAOSVersion object."""
         super().__init__(*args, **kwargs)
-        # Don't waste time starting servers and agents.
-        self.setup_start_servers = False
-        self.setup_start_agents = False
```

Comment:
Was this intentional? I added those lines because I noticed that the test was wasting a lot of time in setting up and then starting the server and agent for no reason. The test doesn't need anything to be running in order to check the version of the utilities.

Reply:
In fact, the configuration files and certificates  seems to not be properly generated when the service are not started.
From what I have understand, the test should not be run by the CI with your PR (as it is not in the pr tag) and locally it should work if these files where previously generated.

Reply:
OK, but my point is that the config/certs shouldn't need to be generated in order to run the version commands. If there is an error when running e.g. 'daos version` without the agent or configuration, then that is a bug that needs to be fixed.

Reply:
In fact, it is dmg which is complaining.

Reply:
The agent could probably be not started, however I suspect that the server has to be started for properly generating the daos_control.yml file.

Reply:
I will make more experiments tomorrow to try to find the best compromise.

Reply:
Ah, then that is a bug that needs to be fixed. None of the commands should ever require a configuration in order to print the version. I would prefer to keep the behavior I added so that we find this more easily if it regresses in the future.

Reply:
OK, I will try to fix in this PR as it should not be so complex.
Is it OK for you ?

Reply:
Sounds good to me, thanks.

Reply:
Fixed with commit a233d3fd050f28a749f98427f7290dbacb66f0b5

Reply:
@mjmac my apologies for the rebase.
I was still working as a draft PR.
Following is the difference between the version that you have reviewed and the latest version.
https://github.com/daos-stack/daos/compare/30411573abd590510ad9b4675dfa122df7865ba7..a233d3fd050f28a749f98427f7290dbacb66f0b5